web/cobrands/fixmystreet/header.js

```diff
@@ -11,7 +11,10 @@ fixmystreet.cobrand = '[% c.cobrand.moniker %]';
     if ('IntersectionObserver' in window) {
         E.className += ' lazyload';
     }
-    if (type == 'mobile') {
-        E.className += ' mobile[% " map-fullscreen only-map map-reporting" IF page == "around" || page == "new" %]';
+    var isShortScreen = Modernizr.mq('(max-height: 30em)');
+    if (!isShortScreen) {
+        if (type === 'mobile') {
+            E.className += ' mobile[% " map-fullscreen only-map map-reporting" IF page == "around" || page == "new" %]';
```

Comment:
This looks much better, but we still want the mobile class, we just don't want the full screen only map. Currently:

| Width | Height | Current live output | Current output with this PR | Desired output |
| --- | --- | --- | --- | --- |
| <48em | <30em | mobile/fullscreen | nothing | mobile |
| <48em | >=30em | mobile/fullscreen | mobile/fullscreen | mobile/fullscreen |
| >=48em | <30em | nothing | nothing | nothing |
| >=48em | >=30em | nothing | nothing | nothing |

I've added a fixup, have a look and see if that works for you.